panopticum/migrations/0004_auto_20200322_1402.py

```diff
@@ -0,0 +1,84 @@
+# Generated by Django 2.1.15 on 2020-03-22 11:02
```

Comment:
Lets squash migrations https://docs.djangoproject.com/en/2.1/topics/migrations/#squashing-migrations
It will allow to easy manage migrations. Lets make one migration per Pull request